### ISSUE TITLE
modify size of training set + increase size of test set

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/same-different-transformers#3]
- we'll want to manipulate training set size (perhaps logarithmically increasing) to see how many examples are required to get OOD generalization
- also, would be useful (if possible!) to have a test set size that is as large as the largest training set size (better ensuring our results are genuinely valid)
